app/src/components/map/MapContainer.tsx

```diff
@@ -87,6 +103,32 @@ const MapContainer: React.FC<IMapContainerProps> = (props) => {
         <LayersControl.BaseLayer name="BC Government">
           <TileLayer url="https://maps.gov.bc.ca/arcgis/rest/services/province/roads_wm/MapServer/tile/{z}/{y}/{x}" />
         </LayersControl.BaseLayer>
+        {hideOverlayLayers !== true && (
```

Comment:
nit
can we just say: 
```suggestion
        {!hideOverlayLayers && (
```

Reply:
fixed